websites/U/Unicode Character Table/presence.ts

```diff
@@ -0,0 +1,179 @@
+const presence = new Presence({
+		clientId: "1024322774782713977",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/oDtbM69.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href, search } = window.location,
+		pathSplit = pathname
+			.split("/")
+			.filter(x => x)
+			.slice(1);
+
+	switch (pathSplit[0] ?? "") {
+		case "": {
+			presenceData.details = "Browsing list of unicode characters";
+			presenceData.state =
+				document.querySelector<HTMLSpanElement>(".box__title").textContent;
+			break;
+		}
+		case "alphabets": {
+			if (pathSplit[1]) {
+				presenceData.details = "Browsing alphabet";
+				presenceData.state =
+					document.querySelector<HTMLSpanElement>("h1").textContent;
+			} else presenceData.details = "Browsing list of alphabets";
+			break;
+		}
+		case "alt-codes": {
+			presenceData.details = "Browsing list of Windows alt codes";
+			break;
+		}
+		case "blocks": {
+			presenceData.details = "Browsing unicode blocks";
+			if (pathSplit[1]) {
+				presenceData.state = `${
+					document.querySelector<HTMLHeadingElement>("h1").textContent
+				} (${
+					document
+						.querySelector<HTMLSpanElement>(".page-content__info-range")
+						.textContent.match(/([A-Z0-9]{4,6}—[A-Z0-9]{4,6})/)[1]
+				})`;
+			}
+			break;
+		}
+		case "emoji": {
+			presenceData.details = "Browsing list of emojis";
+			if (pathSplit[1]) {
+				presenceData.state =
+					document.querySelector<HTMLHeadingElement>("h1").textContent;
+			}
+			break;
+		}
+		case "holidays": {
+			if (pathSplit[1]) {
+				presenceData.details = "Browsing holiday symbols";
+				presenceData.state =
+					document.querySelector<HTMLHeadingElement>("h1").textContent;
+			} else presenceData.details = "Browsing list of holiday sets";
+			break;
+		}
+		case "html-entities": {
+			presenceData.details = "Browsing list of HTML entities";
+			break;
+		}
+		case "kaomoji": {
+			presenceData.details = "Browsing list of kaomoji";
+			break;
+		}
+		case "search": {
+			presenceData.details = "Searching for characters";
+			presenceData.state = new URLSearchParams(search).get("q");
+			break;
+		}
+		case "sets": {
+			presenceData.details = "Browsing symbol sets";
+			if (pathSplit[1]) {
+				presenceData.state =
+					document.querySelector<HTMLHeadingElement>("h1").textContent;
+			}
+			break;
+		}
+		case "text-art": {
+			presenceData.details = "Browsing text art";
+			break;
+		}
```

Comment:
Same here:
![image](https://user-images.githubusercontent.com/42322979/192961192-429ee32d-a571-455c-9c78-deb006ceee97.png)
![image](https://user-images.githubusercontent.com/42322979/192961272-bcc0c656-110b-49dc-81da-3a526c4501a3.png)

`document.querySelector('[class="main-navigation__link main-navigation__link--current"]')?.textContent.trim()`

Reply:
This works for some cases, but there are other cases with other logic where I don't want to simply remove and replace with this.